packages/utils/src/io.ts

```diff
@@ -100,6 +100,7 @@ export interface FetchOptions {
   readonly path: string;
```

Comment:
I'm not clear on why we maintain our own options object here instead of, at most, intersecting with http.RequestOptions.

Reply:
I can’t think of any reason. This seems bad.

Reply:
I'll fix it.